lib/cfnlego.rb

```diff
@@ -0,0 +1,57 @@
+require 'yaml'
+require 'erb'
+require 'ruby-beautify'
+require 'cfnlego/cloudformation'
+require 'cfnlego/resource'
+require 'cfnlego/version'
+require 'net/http'
+require 'uri'
+
+JSON_URI = 'https://d2stg8d246z9di.cloudfront.net/latest/CloudFormationResourceSpecification.json'
```

Comment:
Should this be us-east-1?
https://d1uauaxba7bl26.cloudfront.net/latest/CloudFormationResourceSpecification.json

@kornypoet, no need to get this file twice for example.